bin/resources/GameIndex.yaml

```diff
@@ -39049,6 +39199,7 @@ SLPS-25052:
     alignSprite: 1 # Fixes vertical lines.
     mergeSprite: 1 # Fixes vertical lines.
     gpuTargetCLUT: 1 # Fixes broken sun.
+    gpuPaletteConversion: 2 # Stops excessive VRAM usage with preloading on.
```

Comment:
this isnt needed  also

Reply:
I can remove it but FMVs need to have a fix in the future then, it's not system critical so guess it will be low priority as they don't need that much power